package.json

```diff
@@ -41,7 +45,39 @@
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-prettier": "^3.1.4",
     "jest": "^24.5.0",
-    "prettier": "2.1.2"
+    "prettier": "2.1.2",
+    "release-it": "^14.2.1",
+    "release-it-lerna-changelog": "^3.1.0",
+    "release-it-yarn-workspaces": "^2.0.0"
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org"
+  },
+  "release-it": {
+    "plugins": {
+      "release-it-lerna-changelog": {
+        "infile": "CHANGELOG.md",
+        "launchEditor": true
+      },
+      "release-it-yarn-workspaces": {
+        "workspaces": [
+          "packages/*"
+        ],
+        "additionalManifests": {
+          "dependencyUpdates": [
+            "test-packages/*/package.json"
+          ]
+        }
+      }
+    },
```

Comment:
This configuration is the only real difference from `npx create-rwjblue-release-it-setup`. It is specifically needed because we don't "just" want to use the `workspaces` array directly (since we don't want to publish `test-packages/*`).